web/cypress/e2e/app.cy.ts

```diff
@@ -38,3 +38,27 @@ describe("ARC Portal UI authenticated", () => {
     cy.contains("Agreement confirmed").should("be.visible");
   });
 });
+
+describe("Setting chosen name for user", () => {
+  const chosenName = (Cypress.env("chosenName") as string) || "Test Chosen Name";
+
+  it("prompts user to set chosen name", () => {
+    cy.loginAsAdmin();
+    cy.clearChosenName();
+    cy.visit("/");
+
+    cy.get("dialog[data-id='chosenName']").should("be.visible");
```

Comment:
I think we should be consistent with selectors. above uses `id`s. I'm not adverse to switching everything to `data-id` or `data-cy` (despite @jhughes982 's aversion 😉), but we should be consistent

Reply:
I reckon I'll change this one to `data-cy` because it's only there for cypress since CSS modules means I can't just select by the ID now (the aforementioned peeve)

Reply:
can we change all the `id`-s to `data-cy` not just this one?

Reply:
I'd argue it depends if the id is also being used for styling. in tests, yes let's stick to using data-cy for consistency